static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -88,9 +116,28 @@
     }
 }
 
+.atom-quiz__answer__text .atom-quiz__answer__icon {
+    position: absolute;
+    left: 1em;
+    top: .9em;
+}
+
+.atom-quiz__answer__image .atom-quiz__answer__icon {
```

Comment:
To keep specificity down, it might be worth consolidating these to modifiers, e.g. `atom-quiz__answer__icon--text` and `atom-quiz__answer__icon--image`